misk/src/test/kotlin/misk/hash/ConsistentHashRingTest.kt

```diff
@@ -0,0 +1,39 @@
+package misk.hash
+
+import com.google.common.hash.Hashing
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+
+internal class ConsistentHashRingTest {
+  @Test fun singleNode() {
+    val hashRing = ConsistentHashRing(
+        members = mapOf("zork" to "zork"),
+        hashFn = Hashing.murmur3_32(0))
+    Assertions.assertThat(listOf("foo", "bar", "zed").map { hashRing[it] })
+        .containsExactly("zork", "zork", "zork")
```

Comment:
I kind of want to show this test case to my friends when they ask me what computer programming is